sdk/resourcemanager/azure-resourcemanager-compute/CHANGELOG.md

```diff
@@ -4,11 +4,7 @@
 
 ### Features Added
 
-### Breaking Changes
-
-### Bugs Fixed
-
-### Other Changes
+- Supported `capacityReservation` and `withCapacityReservation` methods for `VirtualMachine`.
```

Comment:
update this

Reply:
Fixed in the [commit#d3df8a2](https://github.com/Azure/azure-sdk-for-java/pull/42914/commits/d3df8a26b67b9430da2e4363e5b7c50ae8bfa316)